app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/LoginUseCase.java

```diff
@@ -166,7 +165,15 @@ private void checkUserCredentialsWithOrgUnit(Credentials credentials,
             if (fromNetWorkError) {
                 notifyNetworkError();
             } else {
-                notifyInvalidCredentials();
+                if (insertedCredentials.getUsername().equals(credentials.getUsername())
```

Comment:
if on the remote server pin for a user is changed then in a push process when validate username and pin previously to the push process,  the app should navigate to login without to do logout but with this code push is blocking but does not navigate to login screen.